### ISSUE TITLE
Bug: APIs related to M2M columns become undefined in Swagger

### ISSUE THREAD

**Issue by wingkwong** [nocodb/nocodb#348]
**Please enter the following details**

APIs related to M2M columns become undefined in Swagger

**Steps To Reproduce**
1. Create a table called ``Table A``
2. Create a table called ``Table B``
3. In ``Table B``, create a M2M column linking to ``Table A``
4. Go to Swagger, there are two called ``TableAHasManyundefined`` and ``TableBHasManyundefined``. 
5. Try executing something like ``/nc/xxxxxxxxx/api/v1/TableB/has/m2mTableA_TableB'`` and got the following result.
```json
{
  "msg": "Cannot read property 'columnToAlias' of undefined"
}
```
6. There are some APIs with undefined id as well 
```
/nc/xxxxxxxxx/api/v1/TableB/{TableBId}/m2mTableA_TableB/{undefinedId}
```

**Expected behavior**
No undefined values

Node: v12.22.1
Arch: x64
Platform: linux
Docker: true
Database: pg
ProjectOnRootDB: false
RootDB: pg
PackageVersion: 0.10.3

**Comment by wingkwong** [comment]
For the ``undefined`` in titles (step 4), I believe that ``${this.ctx.ctn}`` should be changed to ``${this.ctx._ctn}`` instead.

One of the example lines: 
https://github.com/nocodb/nocodb/blob/69be190d68a8c527fbacdeb5b1b16fc9c7df4dcc/packages/nocodb/src/lib/sqlMgr/code/routers/xc-ts/SwaggerXcHm.ts#L65

**Comment by pranavxc** [comment]
Fixed in `v0.10.4`
